pythonforandroid/bootstraps/sdl2/__init__.py

```diff
@@ -100,7 +106,55 @@ def run_distribute(self):
                         shprint(sh.rm, '-f', filename)
                     shprint(sh.rm, '-rf', 'config/python.o')
 
-            else:  # Python *is* loaded from crystax
+            elif self.ctx.python_recipe.name == 'python3':
+                ndk_dir = self.ctx.ndk_dir
+                py_recipe = self.ctx.python_recipe
+
+                ## Build the python bundle:
+
+                # Bundle compiled python modules to a folder
+                modules_dir = join(python_bundle_dir, 'modules')
+                ensure_dir(modules_dir)
+                
+                modules_build_dir = join(
+                    self.ctx.python_recipe.get_build_dir(arch.arch),
+                    'android-build',
+                    'lib.linux-arm-3.7')
+                module_filens = (glob.glob(join(modules_build_dir, '*.so')) +
+                                 glob.glob(join(modules_build_dir, '*.py')))
+                for filen in module_filens:
+                    shprint(sh.cp, filen, modules_dir)
+
+                # zip up the standard library
+                stdlib_zip = join(self.dist_dir, python_bundle_dir, 'stdlib.zip')
+                with current_directory(
+                        join(self.ctx.python_recipe.get_build_dir(arch.arch),
+                             'Lib')):
+                    shprint(sh.zip, '-r', stdlib_zip, *os.listdir())
```

Comment:
Fix missing argument for command os.listdir, we can fix it like this:

```
                stdlib_dir = join(
                    self.ctx.python_recipe.get_build_dir(arch.arch), 'Lib')
                with current_directory(stdlib_dir):
                    shprint(sh.zip, '-r', stdlib_zip, *os.listdir(stdlib_dir))```

Reply:
Alternative fix: drop python2 support :<

Reply:
If you maintain the affected code as it is you will also drop support for python 3 prior to version 3.2 and it would be great that was documented in somewhere: https://docs.python.org/3/library/os.html?highlight=listdir#os.listdir